static/js/containers/CollectionListPage.js

```diff
@@ -87,7 +88,14 @@ export class CollectionListPage extends React.Component<*, void> {
       return null
     }
     if (currentPageData.status === "ERROR") {
-      return <div className="collection-list-page-error">Error!</div>
+      const supportEmail = SETTINGS.support_email_address || ""
+      return (
+        <ErrorMessage>
+          <p>Sorry, we were unable to load the data necessary to process your request. Please reload the page.</p>
```

Comment:
This content is pretty generic, maybe it could be the default in the `ErrorMessage` component itself?  And customizable if we want to display something slightly different on certain pages?

Reply:
@mbertrand  Hmm...I would prefer to keep the `ErrorMessage` component itself simple, rather than have logic in the `ErrorMessage` component to check whether it was given content or not.

Perhaps we can wait to create generic messages until we have another use case for this particular message?